backend/controllers/userMe.js

```diff
@@ -2,9 +2,11 @@ const express = require('express');
 const router = express.Router();
 const elite2Api = require('../elite2Api');
 const asyncMiddleware = require('../middleware/asyncHandler');
+const config = require('../config');
 
 router.get('/', asyncMiddleware(async (req, res) => {
   const response = await elite2Api.currentUser(req, res);
+  response.data.notmEndpointUrl = config.app.notmEndpointUrl;
```

Comment:
As discuss, let change this to use a new /config endpoint. Will approve for now and you can re-work next week.